<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/rec/Радиостанция_Р_853_В2М _КП10.ico</normaloff>
    <normalon>:/rec/Радиостанция_Р_853_В2М _КП10.ico</normalon>
    <disabledoff>:/rec/Радиостанция_Р_853_В2М _КП10.ico</disabledoff>
    <disabledon>:/rec/Радиостанция_Р_853_В2М _КП10.ico</disabledon>
    <activeoff>:/rec/Радиостанция_Р_853_В2М _КП10.ico</activeoff>
    <activeon>:/rec/Радиостанция_Р_853_В2М _КП10.ico</activeon>
    <selectedoff>:/rec/Радиостанция_Р_853_В2М _КП10.ico</selectedoff>
    <selectedon>:/rec/Радиостанция_Р_853_В2М _КП10.ico</selectedon>:/rec/Радиостанция_Р_853_В2М _КП10.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="cB_ComList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_ComOpen">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_config">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Конфигурация</string>
    </property>
   </widget>
   <widget class="QListView" name="listV_F5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>140</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_F5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Режим [F5]:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_F6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Поддиапазон [F6]:</string>
    </property>
   </widget>
   <widget class="QListView" name="listV_F6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>140</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rButt_F0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>241</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Включение аттенюатора (минус 6 Дб) [F9]</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listW_CheckdB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>140</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkB_F1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>минус 16 дБ [F1]</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkB_F2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>минус 8 дБ [F2]</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkB_F3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>минус 4 дБ [F3]</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkB_F4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>минус 2 дБ [F4]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_reset">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Сбросить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_sniffer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tE_sniffer">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сниффер:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_sniffClear">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CAP">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ЦАП [F8]:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sBox_valueCAP">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_EnterCAP">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввод</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5361">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Генератор шума [F7]:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_5361">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Включен</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
